packages/sdk/src/types/relationshipTemplates/requests/GetPeerRelationshipTemplatesRequest.ts

```diff
@@ -3,4 +3,5 @@ export interface GetPeerRelationshipTemplatesRequest {
     expiresAt?: string | string[];
     createdBy?: string | string[];
     maxNumberOfAllocations?: number | number[];
+    forIdentity?: string | string[];
```

Comment:
`string` instead of `string | string[]` should be sufficient as type, because all PeerRelationshipTemplates I have either have `forIdentity` undefined or have specified my own address there, because otherwise I would not have the RelationshipTemplate.

Reply:
That would be an option, the impression of what I have seen in those requests was that we don't think and just allow everything (isOwn has `boolean | boolean[]`). But GetAttributesRequest has almost no arrays? 🤔

Reply:
"Don't think" is a good motto :D For me, both are okay, so throwing away the string array (for the reasons described) or leaving it.

Reply:
IMO forIdentity with string-array is valid here, you could match for different identities in one query for example.

Reply:
> you could match for different identities in one query for example.

Why it should be possible? All PeerTemplates that I have loaded and can get with this function call have `forIdentity` undefined or with my own address (otherwise, I couldn't load them at the first place).